stm32-modules/include/heater-shaker/systemwide.h

```diff
@@ -18,3 +18,5 @@ typedef enum {
     WHITE_AMBER
 } LED_COLOR;
 typedef enum { SOLID_HOT, SOLID_HOLDING, PULSE, MODE_OFF } LED_MODE;
+
+typedef enum { NONE, SHORT, OPEN, OVERCURRENT } HEATPAD_CIRCUIT_ERROR;
```

Comment:
I think it would make sense to prefix these names since they're pretty generic and they're in the global namespace